dpu-cni/pkgs/sriov/sriov.go

```diff
@@ -504,23 +504,22 @@ func (sm *sriovManager) CmdDel(req *cnitypes.PodRequest) error {
 
 	netConf, cRefPath, err := sriovconfig.LoadConfFromCache(req.ContainerId, req.IfName)
 	if err != nil {
-		// If cmdDel() fails, cached netconf is cleaned up by
-		// the followed defer call. However, subsequence calls
-		// of cmdDel() from kubelet fail in a dead loop due to
-		// cached netconf doesn't exist.
-		// Return nil when LoadConfFromCache fails since the rest
-		// of cmdDel() code relies on netconf as input argument
-		// and there is no meaning to continue.
+		// This might happen if the cache has already been cleaned up. Kubelet
```

Comment:
Do we need to clean this up? I think we can leave it as is. It is easier to do a sync with upstream sriov-cni code base.

Reply:
Good point. Do we want to copy things into this repo or do we want to make this a call into the other repo and vendor it in?

Reply:
Hmm tough call... Since I am looking at the SRIOV Env Variable issue. I can also look into this.

Reply:
Crap. I just remembered we had to hack the sriov-cni code because the Intel IPU doesn't support things. So vendoring approach won't work as is.